### PR TITLE
Update installation instruction for single machine

### PR DESCRIPTION
**Pull request by nakabonne**
**What this PR does / why we need it**:
It's because `--tools-dir` and `sshConfigFilePath` will be non-required from the next version

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/docs/operator-manual/piped/installation.md [hunk 1]
```diff
@@ -105,28 +105,24 @@ Most part is identical to the previous way, but some part is slightly different.
   apiVersion: pipecd.dev/v1beta1
   kind: Piped
   spec:
-    projectID: YOUR_PROJECT_ID
-    pipedID: YOUR_PIPED_ID
-    pipedKeyFile: PATH_TO_PIPED_KEY_FILE
+    projectID: {YOUR_PROJECT_ID}
```
Comment (khanhtc1202) [inline docs/content/en/docs/operator-manual/piped/installation.md:108]: to keep all config file syntax consistent, I think all values should wrap in `{ }` or not at this time, so without `{ }` is better to me. We can make it by another PR which change all other configuration files 👀

File: docs/content/en/docs/operator-manual/piped/installation.md [hunk 2]
```diff
@@ -105,28 +105,24 @@ Most part is identical to the previous way, but some part is slightly different.
...
     syncInterval: 1m
   ```
 
 - Start running the `piped`
 
   ``` console
-  ./piped piped \
-  --config-file=PATH_TO_PIPED_CONFIG_FILE \
-  --tools-dir=PATH_TO_TOOLS_DIR
+  ./piped piped --config-file=PATH_TO_PIPED_CONFIG_FILE
```
Comment (khanhtc1202) [inline docs/content/en/docs/operator-manual/piped/installation.md:126]: Nice 😁

File: docs/content/en/docs/operator-manual/piped/installation.md (hunk 1 above)
Comment (nakabonne) [reply docs/content/en/docs/operator-manual/piped/installation.md:108]: >I think all values should wrap in { } or not 

I feel exactly the same way. But I'm guessing the ones enclosed with `{ }` is better because there are a couple of fields to be all capitalized kind of like `KUBERNETES`, it can be confusing.

Comment (khanhtc1202) [reply docs/content/en/docs/operator-manual/piped/installation.md:108]: Sure 👍 Then let's address it with another PR that refine all other configuration files 🙏

Comment (nakabonne) [reply docs/content/en/docs/operator-manual/piped/installation.md:108]: Yep 👍

**Comment by khanhtc1202** [comment]
🚀 
/lgtm

**Review by nghialv — APPROVED** [review]
